App.tsx

```diff
@@ -47,9 +49,18 @@ class App extends React.Component<IProps, IState> {
     }
   }
   async componentDidMount() {
+    NetInfo.fetch().then(async state => {
+      if (state.isConnected && !checkIfDbExists()) {
+        await downloadDB();
+      }
+      await initSchema();
```

Comment:
remove this as it doesnt update the actual config variable, just call it in each load func